### PR TITLE
Fix cursor visibility after leaving visual graph

### PR DESCRIPTION
**Pull request by gur111**
It's a single line that I copied. Tested it's fixing the issue. Think it's enough

### CODE REVIEW

**Comment by radare** [comment]
Screenshot 2020-03-24 at 20 02 50

File: libr/core/agraph.c [hunk 1]
```diff
@@ -4858,6 +4858,7 @@ R_API int r_core_visual_graph(RCore *core, RAGraph *g, RAnalFunction *_fcn, int
 	}
 	r_config_hold_restore (hc);
 	r_config_hold_free (hc);
+	showcursor(core, true);
```
Comment (radare) [inline libr/core/agraph.c]: add missing space, and only do this if color is set

Comment (gur111) [reply libr/core/agraph.c]: Regarding the space, there wasn't a space before either but I'll add it.
Regarding the color. I'm not sure what you mean or how to do that. I merely looked for the place where the cursor is set to hidden and after the graph view closes I added a line to make it visible (it was made invisible earlier in the function).
I can't say I understand the project, merely was able to mimic the existing logic and thought to save someone some time fixing the issue I opened a little before.

**Comment by radare** [comment]
The cursor is only hidden when in visual. So you should only call this from there. This is why it’s my making the testsuite fed fail

**Comment by radare** [comment]
And the space is before the (

File: libr/core/panels.c [hunk 2]
```diff
@@ -7248,6 +7249,9 @@ void __panels_process(RCore *core, RPanels *panels) {
 	}
 	goto repeat;
 exit:
+    if (!originVmode) {
```
Comment (radare) [inline libr/core/panels.c]: use tabs, not spacs

File: libr/core/panels.c [hunk 3]
```diff
@@ -6680,6 +6680,7 @@ void __panels_process(RCore *core, RPanels *panels) {
 
 	bool o_interactive = r_cons_is_interactive ();
 	r_cons_set_interactive (true);
+    r_core_visual_showcursor (core, false);
```
Comment (radare) [inline libr/core/panels.c]: dont indent with spacs

Comment (gur111) [reply libr/core/panels.c]: done

File: libr/core/panels.c (hunk 2 above)
Comment (gur111) [reply libr/core/panels.c]: done

**Pull request merged**